pyccel/codegen/printing/fcode.py

```diff
@@ -599,13 +599,16 @@ def _print_Lambda(self, expr):
 #
 #        return self._get_statement(code)
 
+    def print_kind(self, expr):
+        return iso_c_binding[self._print(expr.dtype)][expr.precision]
+
```

Comment:
Since this function is fundamentally different than the other `_print_TYPE` methods, I think it should be moved to another location.

Reply:
And it needs a docstring!

Reply:
what do you mean by "location" is it in another file or in the same file?.

Reply:
I mean in the same file.
This function cannot be hidden among other `_print_TYPE` methods.
It should be one of the first methods of its class instead.